ip/draft-ietf-detnet-ip.xml

```diff
@@ -935,19 +902,25 @@ DetNet                    |L2/SbN|          |L2/SbN|
             implementations use the first match.  This can, for example, be
             used to provide a DetNet service for a specific UDP flow, with
             unique Source and Destination Port field values, while
-            providing a different service for all other flows with that same
-            UDP Destination Port value.
+            providing a different service for the aggregate of all other
+            flows with that same UDP Destination Port value.
+          </t>
+          <t>
+            It is the responsibility of the DetNet controller plane to
+            properly provision both flow identification information and
+            the flow specific resources needed to provided the traffic
+            treatment needed to meet each flow's service requirements.
+            This applies for aggregated and non-aggregated flows.
```

Comment:
Again terminology: non-aggregated flow = individual flow. Do we need a new term?